cla-backend/cla/utils.py

```diff
@@ -51,11 +67,11 @@ def create_database(conf=None):
     """
     if conf is None:
         conf = cla.conf
-    cla.log.info('Creating CLA database in %s', conf['DATABASE'])
-    if conf['DATABASE'] == 'DynamoDB':
+    cla.log.info("Creating CLA database in %s", conf["DATABASE"])
+    if conf["DATABASE"] == "DynamoDB":
```

Comment:
As far as I know, we use single quotes throughout the codebase. Can you revert the quote changes?